From 76c3bea3c618aa2106ae753cc3e8491dd845089c Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Fri, 21 Jun 2024 14:26:21 +0530
Subject: [PATCH] [WA] Disabled sepolicy and vndk in A15.

Disabling sepolicy and vndk for now.

Tests: Prepared EB successfully.

Tracked-On: NA
Change-Id: Ibc971eb16c788733267d09f0f645261d55a77974
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 compatibility_matrix.xml | 4 ++--
 manifest.xml             | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/compatibility_matrix.xml b/compatibility_matrix.xml
index 41ada30..a0bf31f 100644
--- a/compatibility_matrix.xml
+++ b/compatibility_matrix.xml
@@ -47,7 +47,7 @@
             <instance>default</instance>
         </interface>
     </hal>-->
-    <vendor-ndk>
+    <!--<vendor-ndk>
         <version>34</version>
-    </vendor-ndk>
+    </vendor-ndk>-->
 </compatibility-matrix>
diff --git a/manifest.xml b/manifest.xml
index bc2ccc9..96f1ba1 100644
--- a/manifest.xml
+++ b/manifest.xml
@@ -55,7 +55,7 @@
         </interface>
     </hal>
     <!--<kernel target-level="5"/>-->
-    <sepolicy>
+    <!--<sepolicy>
         <version>34.0</version>
-    </sepolicy>
+    </sepolicy>-->
 </manifest>
-- 
2.34.1

